### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by smudge202**
Just some more cleanup tasks.

I've refrained from doing anything serious with the new Text Resources, but they hella mess right now. Every resource lookup causes a fresh seed (`GetResources`), which is then culture filtered (`GetResourcesByCulture`), then finally converted into a List, then a Dictionary (you didn't commit the last minute removal of `ToList` yet), which is finally hash compared for the correct resource. I'm all for avoiding pre-optimisation, but wow! :D :P

I'll see about adding issues for patterns/ideas/etc.

### CODE REVIEW

**Comment by ariugwu** [comment]
Ha! Yes. The Text Resources currently will cause skin to crawl a little. First step to cleaning out a closet is to get everything out and onto the floor. ;)

Once everything is out and we can see if the general approach is solid I think the optimizations happen very quickly. I think the convert to Dictionary should be the onus of the hive mind.

Not to mention that the model and enum should be in the data project or we're going to hit a circular reference. >_>

During the stream I try to refrain from jumping around _too_ much. 

tl;dr - Stream coding is hard. :D

**Comment by smudge202** [comment]
Yeh, I understand developing whilst streaming can be difficult. Had a crack at it myself not too long ago (https://www.youtube.com/playlist?list=PLDi7RpmhhWQnXyA_SeTKlF83eWpWO7z-0). For me, the best parts of my streams was when I was collaborating because the guys would get talking and I could crack on with something or make some decisions in relative piece before continuing.

You're doing good though. The audience is learning, I'm learning, you're learning. Everyone's learning. :)
